bin/transmux.js

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const path = require('path');
+const {Transmuxer} = require('../lib/mp4');
+const {version} = require('../package.json');
+const {concatTypedArrays, bytesToString} = require('@videojs/vhs-utils/cjs/byte-helpers');
+
+const showHelp = function() {
+  console.log(`
+  transmux media-file > foo.mp4
+  transmux media-file -o foo.mp4
+  curl -s 'some-media-ulr' | transmux.js -o foo.mp4
+  wget -O - -o /dev/null 'some-media-url' | transmux.js -o foo.mp4
+
+  transmux a supported segment (ts or adts) info an fmp4
+
+  -h, --help                 print help
+  -v, --version              print the version
+  -o, --output    <string>   write to a file instead of stdout
+  -d, --debugger             add a break point just before data goes to transmuxer
+`);
+};
+
+const parseArgs = function(args) {
+  const options = {};
+
+  for (let i = 0; i < args.length; i++) {
+    const arg = args[i];
+
+    if ((/^--version|-v$/).test(arg)) {
+      console.log(`transmux.js v${version}`);
+      process.exit(0);
+    } else if ((/^--help|-h$/).test(arg)) {
+      showHelp();
+      process.exit(0);
+    } else if ((/^--debugger|-d$/).test(arg)) {
```

Comment:
we currently error out if `muxjs-transmux` is run without arguments, we probably should default to `showHelp()`